### ISSUE TITLE
eos_token_id`:57001  模型输出内容出现大量重复单词

### ISSUE THREAD

**Issue by hotmengmeng** [HIT-SCIR/huozi#14]
报错内容：
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:57001 for open-end generation.

导致现象：回答的内容中会一直重复一些内容，如一个词

我是使用Python调用的awq模型

请问各位大大错误是是什么原因呀？

**Comment by jubgjf** [comment]
您好，可以提供下环境版本吗

**Comment by hotmengmeng** [comment]
> 您好，可以提供下环境版本吗

我是用的AutoDL的服务器，GPU-L20(48GB) * 1
pytorch2.1.0，
Python 3.10.8，
ubuntu 22.04，
cuda12.1，
accelerate      0.30.1，
autoawq        0.2.5
autoawq-kernels        0.0.6+cu121
transformers                   4.41.1
我改了好久 ，实在弄不好，麻烦了，谢谢！

**Comment by jubgjf** [comment]
量化模型确实可能发生重复输出的问题：https://github.com/THUDM/ChatGLM3/issues/141

如果AWQ量化出问题，可以尝试调整temperature/top-p/top-k等参数，或者尝试一下llama.cpp
